schema.sql

```diff
@@ -0,0 +1,12 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+	id INT GENERATED ALWAYS AS IDENTITY NOT NULL,
+	name varchar(100) NOT NULL,
+	date_of_birth date NOT NULL,
+	escape_attempts INT NOT NULL,
+	neutered boolean NOT NULL,
+	weight_kg decimal NOT NULL,
+	species varchar(50) NOT NULL,
```

Comment:
- [ ] From the project requirement, I can see clearly that species was not part of the column specified to create an animals table. so I think you should remove it to avoid any future bugs 👍🏽